Makefile

```diff
@@ -268,6 +286,7 @@ install_data : $(DATA_INSTALL_DIR) $(DATA_SRC_DIR)
 ifdef SUDO_USER
 ifeq ("$(USER)","root")
 	chown -R ${SUDO_USER}:${SUDO_USER} $(INSTALL_DIR)/$(DATA_SRC_DIR)
```

Comment:
Why don't we just run chown on the INSTALL_DIR. Then we wouldn't need multiple steps. @izoratti 
 is there a reason to specify the DATA_SRC_DIR rather than the general INSTALL_DIR?